tcms/testplans/templates/testplans/get.html

```diff
@@ -359,6 +369,7 @@ <h2 class="card-pf-title">
     <li data-filter-type="reviewer"><a>{% trans 'Reviewer' %}</a></li>
 </template>
 
+<script src="{% static 'sortablejs/dist/sortable.umd.js' %}"></script>
```

Comment:
Is this reference to node_modules folder? If it is, Im not sure if its good practice, maybe we should place the library in ../tcms/static/js/lib

Reply:
node_modules is the same as static. We've been using it like that for over 2 years.